spec/intergration/post_index_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+require 'capybara/rspec'
+
+RSpec.describe Post, type: :system do
+  user = User.create(name: 'Tom', posts_counter: 3, photo: 'https://pic.com',
+                     bio: 'Project manager')
+  subject do
+    Post.new(author_id: user.id, title: 'Post 2', text: 'Text for Post 2', comments_counter: 2, likes_counter: 2)
+  end
+  before { subject.save }
+  describe 'Post index page' do
+    it "I can see the user's profile picture." do
+      visit user_posts_path(user.id)
+      page.has_css?('img')
+    end
+
+    it "I can see the user's username." do
+      visit user_posts_path(user.id)
+      page.has_content?(user.name)
+    end
+
+    it 'I can see the number of posts the user has written.' do
+      visit user_posts_path(user.id)
+      page.has_content?(user.posts_counter)
+    end
+
+    it "I can see a post's title." do
+      visit user_posts_path(user.id)
+      page.has_content?(subject.title)
+    end
```

Comment:
You did a pretty solid job with the test you have written so far, however, you skipped one of the tests that asserts you can see a section for pagination if there are more posts than fit on the view, consider including this integration in order to satisfy the project requirements